px4_ros2_cpp/include/px4_ros2/control/setpoint_types/fixedwing/lateral_longitudinal.hpp

```diff
@@ -204,6 +206,15 @@ struct FwControlConfiguration
     target_climb_rate = target_climb_rate_sp;
     return *this;
   }
+
+  FwControlConfiguration & withSpeedWeight(float speed_weight_sp)
```

Comment:
Can you add some API docs? Just seeing this it would not be clear to me what it does, and what values to use.